src/core/stream_send.c

```diff
@@ -378,6 +378,10 @@ QuicStreamCompleteSendRequest(
             !(Stream->SendBufferBookmark->Flags & QUIC_SEND_FLAG_BUFFERED));
     }
 
+    if (SendRequest->Flags & QUIC_SEND_FLAG_START && !Stream->Flags.Started) {
```

Comment:
We probably need to de-dup multiple send calls with the `QUIC_SEND_FLAG_START` flag.